src/python/doc/installation.rst

```diff
@@ -41,8 +41,9 @@ there.
 
```

Comment:
We should mention (here and in the relevant section below) that building the doc directly from a git checkout does not work (or make it work...).

Reply:
As it is a legacy behaviour, I created #1234 for not to forgot this issue. It is also the case for the C++ part